rsql_drivers/src/connection.rs

```diff
@@ -81,6 +85,14 @@ impl QueryResult for MemoryQueryResult {
     }
 }
 
+#[derive(Debug, Clone)]
+pub enum QueryMeta {
```

Comment:
A few thoughts:

- Would this be better named `Sql...` or `Statement...` since DML is SQL/Statement, but not a query?
- Should `Meta` be spelled out to `Metadata` to align with the existing `Metadata` struct?
- Should the existing `Metadata` struct be renamed to something like `DatabaseMetadata`?

Reply:
I'll say yes on the first two. For the third question, I think either name would work.